src/adaptors/http2/http2_adaptor.c

```diff
@@ -2972,7 +2972,9 @@ static void egress_conn_timer_handler(void *context)
         // It is ok to call qdr_connection_closed from this timer callback.
         //
         sys_mutex_unlock(qd_server_get_activation_lock(http2_adaptor->core->qd->server));
-        qdr_connection_closed(conn->qdr_conn);
+        if (conn->qdr_conn) {
+            qdr_connection_closed(conn->qdr_conn);
+        }
```

Comment:
Extra safe: set conn->qdr_conn = 0. 
Calling qdr_connection_closed() will free that qdr_conn and since "conn" can live beyond that point qdr_conn will be an invalid pointer.

Reply:
> Extra safe: set conn->qdr_conn = 0. Calling qdr_connection_closed() will free that qdr_conn and since "conn" can live beyond that point qdr_conn will be an invalid pointer.

Done ! Thanks much.